avaframe/out3Plot/outAB.py

```diff
@@ -279,8 +283,16 @@ def WriteResults(eqOutput, saveOutPath):
         outfile.write('Alpha Beta AlMinus1SD AlMinus2SD AlPlus1SD\n')
         outfile.write('{:<10s} {:<10s} {:<10s} {:<10s} {:<10s} \n'.format(
             'x', 'y', 'z', 's', 'angle'))
-        outfile.write('{:<10.2f} {:<10.2f} {:<10.2f} {:<10.2f} {:<10.2f} \n'.format(
-            x[ids_alpha], y[ids_alpha], z[ids_alpha], s[ids_alpha], alpha))
+        if ids_alpha:
+            outfile.write('{:<10.2f} {:<10.2f} {:<10.2f} {:<10.2f} {:<10.2f} \n'.format(
+                x[ids_alpha], y[ids_alpha], z[ids_alpha], s[ids_alpha], alpha))
+        else:
+            outfile.write('{:<10.2f} {:<10.2f} {:<10.2f} {:<10.2f} {:<10.2f} \n'.format(
```

Comment:
Same as above, use the string multiplier 
https://stackoverflow.com/a/17278762/13011708